tests/ui/test_signposting_ui.py

```diff
@@ -9,25 +9,73 @@
 
 See https://signposting.org/FAIR/#level2 for more information on Signposting
 """
+import pytest
 
 
-def test_link_in_landing_page_response_headers(running_app, client, record):
-    res = client.head(f"/records/{record.id}")
+@pytest.mark.parametrize("http_method", ["head", "get"])
+def test_link_in_landing_page_response_headers(
+    running_app, client, record_with_file, http_method
+):
+    ui_url = f"https://127.0.0.1:5000/records/{record_with_file.id}"
+    api_url = f"https://127.0.0.1:5000/api/records/{record_with_file.id}"
+    filename = "article.txt"
 
-    assert (
-        res.headers["Link"]
-        == f'<https://127.0.0.1:5000/api/records/{record.id}> ; rel="linkset" ; type="application/linkset+json"'  # noqa
-    )
+    client_http_method = getattr(client, http_method)
+    res = client_http_method(f"/records/{record_with_file.id}")
 
+    assert res.headers["Link"].split(" , ") == [
+        # The test record does not have an author with an identifier.
+        # The test record does not have a cite-as since it has no DOI.
+        f'<{api_url}> ; rel="describedby" ; type="application/dcat+xml"',
+        f'<{api_url}> ; rel="describedby" ; type="application/json"',
+        f'<{api_url}> ; rel="describedby" ; type="application/ld+json"',
+        f'<{api_url}> ; rel="describedby" ; type="application/marcxml+xml"',
+        f'<{api_url}> ; rel="describedby" ; type="application/vnd.citationstyles.csl+json"',
+        f'<{api_url}> ; rel="describedby" ; type="application/vnd.datacite.datacite+json"',
+        f'<{api_url}> ; rel="describedby" ; type="application/vnd.datacite.datacite+xml"',
+        f'<{api_url}> ; rel="describedby" ; type="application/vnd.geo+json"',
+        f'<{api_url}> ; rel="describedby" ; type="application/vnd.inveniordm.v1+json"',
+        f'<{api_url}> ; rel="describedby" ; type="application/vnd.inveniordm.v1.full+csv"',
+        f'<{api_url}> ; rel="describedby" ; type="application/vnd.inveniordm.v1.simple+csv"',
+        f'<{api_url}> ; rel="describedby" ; type="application/x-bibtex"',
+        f'<{api_url}> ; rel="describedby" ; type="application/x-dc+xml"',
+        f'<{api_url}> ; rel="describedby" ; type="text/x-bibliography"',
+        f'<{ui_url}/files/{filename}> ; rel="item" ; type="text/plain"',
+        # The test record does not have a license.
+        '<https://schema.org/Photograph> ; rel="type"',
+        '<https://schema.org/AboutPage> ; rel="type"',
+        f'<{api_url}> ; rel="linkset" ; type="application/linkset+json"',
```

Comment:
The logic for the landing page is implemented in `FAIRSignpostingProfileLvl1Serializer` in `invenio-rdm-records` and is already tested there (see inveniosoftware/invenio-rdm-records#1908).
It stills makes sense to at least issue the HTTP call to the endpoint here, to make sure that the decorator is working properly, but maybe the assertion should be less detailed to avoid having to adapt this test every time we modify the other module?